### PR TITLE
Update locked_out.md

### PR DESCRIPTION
**Pull request by kenguest**
I forgot my password, while still logged in, found out how to reset it by creating another account... 
I've updated relevant parts of this document to detail the process I followed, and to also explicitly note how to reset another user's password and how to delete a user.

Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [x ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

### CODE REVIEW

File: source/_docs/locked_out.md [hunk 1]
```diff
@@ -10,16 +10,33 @@ or need to recover your data.
...
+If you’ve forgotten your username, then deleting the files mentioned above will be necessary to start a new onboarding process.
 
-If you know the user, but not the password and you can access the [Home Assistant console](https://www.home-assistant.io/hassio/commandline/) and use the command below:
+If you know the username, but not the password and you can access the [Home Assistant console](https://www.home-assistant.io/hassio/commandline/) and use the command below:
 
 Connect a keyboard and monitor to your device.
 
 `auth reset --username existing_user --password new_password`
 
+#### To reset a user's password, as an administrator
```
Comment (frenck) [inline source/_docs/locked_out.md]: What is unclear right now, is is this paragraph needed if I did the auth reset listed above?

Maybe adding an additional introduction in this paragraph helps? Also, if listing steps, please use a type of list to make clear it is a step-by-step instruction that is given.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_docs/locked_out.md (hunk 1 above)
Comment (kenguest) [reply source/_docs/locked_out.md]: You're absolutely correct - what I added is to do with actions done through the UI. I'll explicitly state that later.

**Review by kenguest — COMMENTED** [review]
hi @frenck - that's done now. please let me know if there's anything else I need to tweak. :-)

File: source/_docs/locked_out.md [hunk 2]
```diff
@@ -10,16 +10,34 @@ or need to recover your data.
...
 
-If you know the user, but not the password and you can access the [Home Assistant console](https://www.home-assistant.io/hassio/commandline/) and use the command below:
+#### To reset a user's password, via console
+If you know the username, but not the password and you can access the [Home Assistant console](https://www.home-assistant.io/hassio/commandline/) and use the command below:
 
 Connect a keyboard and monitor to your device.
 
 `auth reset --username existing_user --password new_password`
 
+#### To reset a user's password, as an administrator via the web interface
```
Comment (frenck) [inline source/_docs/locked_out.md:24]:
```suggestion
#### To reset a user's password, as an administrator via the web interface

```

File: source/_docs/locked_out.md [hunk 3]
```diff
@@ -10,16 +10,34 @@ or need to recover your data.
...
+1. Click "Configuration" in the sidebar.
+2. Scroll down to "Users", and click in to that section.
+3. Click on the user, this should bring up a dialog box.
+4. At the bottom of the dialog box, click "Change Password".
+5. Enter the new password, and then click "OK".
+6. Confirm the new password by entering it again, and then click "OK" again.
+7. A confirmation box will be displayed with the text "Password was changed successfully".
+
+#### To delete a user, as an administrator via the web interface
```
Comment (frenck) [inline source/_docs/locked_out.md:34]:
```suggestion
#### To delete a user, as an administrator via the web interface

```

**Comment by homeassistant** [comment]
Hi @kenguest, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16037).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by homeassistant** [comment]
Hi @kenguest, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16037).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by frenck — APPROVED** [review]
Thanks, @kenguest 👍

**Pull request merged**